Sources/Sentry/SentryCrashIntegration.m

```diff
@@ -98,11 +98,18 @@ - (void)installWithOptions:(nonnull SentryOptions *)options
 - (void)startCrashHandler
 {
     void (^block)(void) = ^{
-        SentryFrameInAppLogic *frameInAppLogic =
-            [[SentryFrameInAppLogic alloc] initWithInAppIncludes:self.options.inAppIncludes
-                                                   inAppExcludes:self.options.inAppExcludes];
-        installation =
-            [[SentryCrashInstallationReporter alloc] initWithFrameInAppLogic:frameInAppLogic];
+        BOOL isInstallationNil = NO;
+        if (installation == nil) {
+            SentryFrameInAppLogic *frameInAppLogic =
+                [[SentryFrameInAppLogic alloc] initWithInAppIncludes:self.options.inAppIncludes
+                                                       inAppExcludes:self.options.inAppExcludes];
+
+            installation =
+                [[SentryCrashInstallationReporter alloc] initWithFrameInAppLogic:frameInAppLogic];
+
+            isInstallationNil = YES;
```

Comment:
`installation` just got a new instance and it's not `nil` anymore, but the flag is called `isInstallationNil=yes`, that's odd.
maybe `wasInstallationNil` then?

Reply:
this is a guard to call `sendAllReports`, so maybe we can even use a better name, eg, `isInstalled`, or `canSendReports`

Reply:
I agree. I renamed it thanks.